packages/connect/src/relay/index.ts

```diff
@@ -1,24 +1,31 @@
-import type { default as LibP2P, MuxedStream, HandlerProps } from 'libp2p'
```

Comment:
API changes + implement `Startable`, `Initializable` and `ConnectInitializable` API

Reply:
Simplify `upgradeInbound` + `upgradeOutbound` method